stylelint-polaris/index.js

```diff
@@ -154,16 +133,10 @@ const stylelintPolarisCoverageOptions = {
       },
       'polaris/global-disallowed-list': [
         // Legacy mixin map-get data
-        /\$typography-condensed/,
-        /\$typography-condensed/,
         /\$base-font-size/,
         /\$line-height-data/,
-        /\$font-family-data/,
         /\$font-size-data/,
         /\$default-browser-font-size/,
-        // Legacy custom properties
-        /--p-button-font/,
-        /--p-badge-font/,
```

Comment:
I think these were supposed to be `--p-button-font-weight` and `--p-badge-font-weight`. I checked both for web and they don't exist.